Model/Config/Source/Available/Available.php

```diff
@@ -116,12 +160,21 @@ protected function configureSDK()
     protected function getPaymentOptionId()
     {
         $method = $this->_paymentmethodFactory->create($this->_class);
-        if($method instanceof PaymentMethod){
+        if ($method instanceof PaymentMethod) {
             return $method->getPaymentOptionId();
         }
         return null;
     }
 
+    protected function getPaymentOptionCode()
+    {
+        $method = $this->_paymentmethodFactory->create($this->_class);
```

Comment:
Dit gebeurd nu op 2 plaatsen binnen deze Class, kun je hier een algemene functie voor maken die door getPaymentOptionCode en getPaymentOptionId wordt gebruikt? Deze sla je op in $this->currentMethod